src/Style/Colour.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PhpSchool\CliMenu\Style;
```

Comment:
Can we PR this separately please? I'm finding it pretty impossible to review this and see the bigger picture. So trying to break it up.

Reply:
You're right - since I removed all color styling from the item styles it doesn't make sense right now.